modules/custom-status/rest/user-lookup-endpoint.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * class UserLookupEndpoint
+ * REST endpoint for searching users
+ */
+
+namespace VIPWorkflow\Modules\CustomStatus\REST;
+
+use WP_REST_Request;
+use WP_User_Query;
+
+defined( 'ABSPATH' ) || exit;
+
+class UserLookupEndpoint {
```

Comment:
We can replace this whole thing with a local call within JS itself using `wp`.

```javascript
wp.apiFetch( { path: '/wp/v2/users?search=test' } )
```

That'll give back the list of matches, and each entry has the entire user object. This way it would save a roundtrip and would be nice and speedy.

Reply:
Good thinking! I completely forgot about the native endpoint.